data/certs/README.txt

```diff
@@ -0,0 +1,4 @@
+The certificate domain.cert and private key domain.key for test only!
```

Comment:
Could you add the `.gitignore` file(or add items to it if exists) to remove this crt/key file checkins. 
We should better add a script under scripts folder to generate the cert/key file dynamically instead of putting the 2 files there and saying "for test only".

Reply:
I can remove the example certificates and key, users will know to create cert and key under this directory. However, there is no need to spend time on scripting and maintaining a script to create cert and key. No better script then Linux `openssl` command.